src/main/java/com/conveyal/taui/controllers/BundleController.java

```diff
@@ -204,11 +219,59 @@ public static Bundle update (Request req, Response res) throws IOException {
         return Persistence.bundles.updateFromJSONRequest(req);
     }
 
-    public static Object getBundle (Request req, Response res) {
-        return Persistence.bundles.findByIdFromRequestIfPermitted(req);
+    public static Bundle getBundle (Request req, Response res) {
+        Bundle bundle = Persistence.bundles.findByIdFromRequestIfPermitted(req);
+
+        // Old bundles did not have these set.
```

Comment:
Might be worth spelling this out a little more - "Older bundles will not have service dates set because this feature was added on [date]". Maybe mention that the intent is to progressively update them as they're loaded.